source/gui/startupDialogs.py

```diff
@@ -153,9 +158,24 @@ def __init__(self, parent):
 		groupLabel = _("License Agreement")
 		sizer = wx.StaticBoxSizer(wx.VERTICAL, self, label=groupLabel)
 		sHelper.addItem(sizer)
-		licenseTextCtrl = wx.TextCtrl(self, size=(500, 400), style=wx.TE_MULTILINE | wx.TE_READONLY | wx.TE_RICH)
+		# Create a fake text control to determine appropriate width of license text box
+		_fakeTextCtrl = wx.StaticText(
+			self,
+			label="a" * 80,  # The GPL2 text of copying.txt wraps sentences at 80 characters
+		)
+		widthOfLicenseText = _fakeTextCtrl.Size[0]
+		_fakeTextCtrl.Destroy()
```

Comment:
This could be put in a helper function, wouldn't want to accidentally use `_fakeTextCtrl` after `Destroy()`.

Reply:
fixed this up